textproto/mod.ts

```diff
@@ -31,9 +28,8 @@ export class TextProtoReader {
    * eliding the final \n or \r\n from the returned string.
    */
   async readLine(): Promise<string | null> {
-    const s = await this.readLineSlice();
-    if (s === null) return null;
-    return str(s);
+    const s: Uint8Array | null = await this.readLineSlice();
```

Comment:
I can change it if this is part of this repository's requirements. As a general rule I never use type inferring as it makes code difficult to read and understand for people unfamiliar with the code base. This is especially true in this case where the return type isn't obvious when reading the method's name. I would agree with you for the other cases of inferred types, but not this one, or at least not with this method name.

Reply:
We typically let the type checker infer the types unless there's a specific reason not to, which isn't the case here

Reply:
I changed it.

Reply:
Regarding tests, they are purely written in TypeScript, they do not use Deno.test as I am unfamiliar with it. I can submit them in another PR as it may take time to convert them.